src/pygef/cpt.py

```diff
@@ -140,6 +141,7 @@ class CPTData:
     delivered_vertical_position_datum: VerticalDatumClass
     delivered_vertical_position_reference_point: str
     column_void_mapping: dict | None
+    headers: dict | None
```

Comment:
```suggestion
    raw_headers: dict
```

Also, I think naming it `raw_headers` makes it clearer to the user that they haven't been processed (beyond being split in arrays).

Reply:
Great suggestion, done.